src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
I don't think this import is being used so could be it be removed please?

Reply:
Well spotted! 9b105e42d5c272c14706e62e348886156d33590d